src/views/partials/go-back.ejs

```diff
@@ -0,0 +1,11 @@
+<div class="card-button-container fr-mt-2w">
+  <span>
+    <button
+      class="fr-btn fr-btn--tertiary fr-icon-arrow-go-back-line fr-btn--icon-left go-back-link"
+    >
+      retour
+    </button>
+  </span>
+  <span></span>
+</div>
+<script type="module" src="<%= js('go-back.js') %>"></script>
```

Comment:
todo : no new line at end of file 😨 Plz configure your IDE to add a new line at end of file on save.